package.json

```diff
@@ -41,13 +41,13 @@
     "test:e2e.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-e2e env-cmd -f ./hot.config.js webpack --hide-modules helpers=./test/helpers/index.js e2e=./test/e2e/index.js",
     "test:e2e.puppeteer": "node test/scripts/run-puppeteer.js test/E2ERunner.html",
     "test:unit": "cross-env-shell npm_config_testPathPattern=. BABEL_ENV=commonjs env-cmd -f ./hot.config.js jest --testPathPattern=$npm_config_testPathPattern",
-    "test:unit.watch": "cross-env-shell BABEL_ENV=commonjs env-cmd -f ./hot.config.js jest --testPathPattern=$npm_config_testPathPattern --watch",
+    "test:unit.watch": "cross-env-shell BABEL_ENV=commonjs env-cmd -f ./hot.config.js jest --watch",
     "test:mobile.dump": "cross-env-shell BABEL_ENV=commonjs_e2e NODE_ENV=test-mobile env-cmd --no-override -f ./hot.config.js webpack --hide-modules helpers=./test/helpers/index.js mobile=./test/e2e/mobile/index.js",
     "test:types": "tsc -p ./test/types",
     "watch": "cross-env-shell BABEL_ENV=commonjs NODE_ENV=watch env-cmd -f ./hot.config.js webpack --hide-modules --watch src/index.js",
     "build": "npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd.min && npm run build:languages && npm run build:languages.min",
-    "build:commonjs": "cross-env-shell BABEL_ENV=commonjs_dist env-cmd -f ./hot.config.js babel src --out-dir commonjs",
-    "build:es": "cross-env-shell BABEL_ENV=es env-cmd -f ./hot.config.js babel src --out-dir es",
+    "build:commonjs": "rimraf ./commonjs && cross-env-shell BABEL_ENV=commonjs_dist env-cmd -f ./hot.config.js babel src --out-dir commonjs --copy-files --no-copy-ignored",
```

Comment:
After bundling to the `es` format, I've spotted that the es directory size has increased by about ~35%. It seems that mostly the 3rdparty folder causes that change. Generally, it affects the `npm pack` results.

Reply:
Thanks! I didn't notice `/es` and `/commonjs` directories contain `*.spec.js` and other unnecessary files. I'll revise it 👍